prime-router/src/main/kotlin/Element.kt

```diff
@@ -792,6 +785,19 @@ data class Element(
         }
     }
 
+    /**
+     * Validate phone number and return pair of region and phone number
+     * @param [phoneNumber] phone numner to validate.
+     * @return region, phone number if valid.  Otherwise, return null, null.
+     */
+    fun isValidatePhoneNumber(phoneNumber: String): Pair<String?, Phonenumber.PhoneNumber?> {
+        val phone = phoneNumberUtil.parse(phoneNumber, "US")
```

Comment:
@oslynn we should probably be okay with parsing Canadian and Mexican phone numbers as well. Is there a way for you to parse for both of those regions and fill in elements 6 and 7 as well?